### PR TITLE
Attach a note | Show note image | Load from DB

### PR DESCRIPTION
**Pull request by LinX64**
This is the first implementation for showing the note image on the note.

### CODE REVIEW

**Comment by LinX64** [comment]
[Attach a Note: Showing Notes Images](https://trello.com/c/hVFdu2at/1766-attach-a-note-showing-notes-images)

File: app/src/main/res/layout/add_note_bottom_sheet.xml [hunk 1]
```diff
@@ -110,11 +110,11 @@
                     <com.google.android.material.textfield.TextInputEditText
                         android:id="@+id/note_input"
                         android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
+                        android:layout_height="150dp"
```
Comment (xameYeoh) [inline app/src/main/res/layout/add_note_bottom_sheet.xml:113]: I am not sure about the context, is there no way to make it non-fixed size here? If so, I would add some dimen resource, or maybe there is already one with similar size for the purpose

File: app/src/main/res/layout/edit_note_bottom_sheet.xml [hunk 2]
```diff
@@ -37,140 +40,152 @@
...
                     android:layout_height="wrap_content"
-                    android:text="You can edit your note here"
-                    android:textAppearance="@style/TextAppearance.Aircasting.Body1"
                     android:layout_marginStart="@dimen/keyline_6"
+                    android:layout_marginTop="@dimen/keyline_2"
                     android:layout_marginEnd="@dimen/keyline_6"
-                    android:layout_marginTop="@dimen/keyline_6"
-                    app:layout_constraintTop_toBottomOf="@id/title"
-                    app:layout_constraintBottom_toTopOf="@id/note_description"
+                    android:text="You can edit your note here"
```
Comment (xameYeoh) [inline app/src/main/res/layout/edit_note_bottom_sheet.xml:85]: I guess we could add string resource for that

**Review by xameYeoh — APPROVED** [review]

File: app/src/main/res/layout/edit_note_bottom_sheet.xml (hunk 2 above)
Comment (LinX64) [reply app/src/main/res/layout/edit_note_bottom_sheet.xml]: Surprisingly, we didn't have translations for the edit note part.

**Pull request merged**